<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LinkGameGUI</class>
 <widget class="QMainWindow" name="LinkGameGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>LinkGameGUI</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGraphicsView" name="View">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>625</width>
      <height>450</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Background">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="TimeBar">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>480</y>
      <width>750</width>
      <height>30</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="maximum">
     <number>0</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
    <property name="invertedAppearance">
     <bool>true</bool>
    </property>
    <property name="format">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Level">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>80</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Score">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>150</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="LevelNum">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>80</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="ScoreNum">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>200</y>
      <width>150</width>
      <height>25</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="ShuffleNum">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>270</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="HintNum">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>350</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Shuffle">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>260</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Hint">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>340</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Shade">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>Background</zorder>
   <zorder>View</zorder>
   <zorder>TimeBar</zorder>
   <zorder>Level</zorder>
   <zorder>Score</zorder>
   <zorder>LevelNum</zorder>
   <zorder>ScoreNum</zorder>
   <zorder>ShuffleNum</zorder>
   <zorder>HintNum</zorder>
   <zorder>Shuffle</zorder>
   <zorder>Hint</zorder>
   <zorder>Shade</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuGame">
    <property name="title">
     <string>Game</string>
    </property>
    <addaction name="actionNew_Game"/>
    <addaction name="actionSave"/>
    <addaction name="actionLoad"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuControl">
    <property name="title">
     <string>Control</string>
    </property>
    <widget class="QMenu" name="menuLevel">
     <property name="title">
      <string>Level</string>
     </property>
     <addaction name="actionLevel1"/>
     <addaction name="actionLevel2"/>
     <addaction name="actionLevel3"/>
     <addaction name="actionLevel4"/>
     <addaction name="actionLevel5"/>
    </widget>
    <addaction name="actionPause"/>
    <addaction name="actionShuffle"/>
    <addaction name="actionHint"/>
    <addaction name="menuLevel"/>
   </widget>
   <widget class="QMenu" name="menuSetting">
    <property name="title">
     <string>Setting</string>
    </property>
    <widget class="QMenu" name="menuBackground">
     <property name="title">
      <string>Background</string>
     </property>
     <addaction name="actionBackground1"/>
     <addaction name="actionBackground2"/>
     <addaction name="actionBackground3"/>
    </widget>
    <widget class="QMenu" name="menuIcons">
     <property name="title">
      <string>Icons</string>
     </property>
     <addaction name="actionIcon1"/>
     <addaction name="actionIcon2"/>
     <addaction name="actionIcon3"/>
    </widget>
    <widget class="QMenu" name="menuMusic">
     <property name="title">
      <string>Music</string>
     </property>
     <addaction name="actionMusic1"/>
     <addaction name="actionMusic2"/>
     <addaction name="actionMusic3"/>
     <addaction name="actionMute"/>
    </widget>
    <addaction name="menuBackground"/>
    <addaction name="menuIcons"/>
    <addaction name="menuMusic"/>
    <addaction name="actionRank"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionInstruction"/>
    <addaction name="actionInformation"/>
   </widget>
   <addaction name="menuGame"/>
   <addaction name="menuControl"/>
   <addaction name="menuSetting"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew_Game">
   <property name="text">
    <string>New Game</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
   <property name="shortcut">
    <string>P</string>
   </property>
  </action>
  <action name="actionShuffle">
   <property name="text">
    <string>Shuffle</string>
   </property>
   <property name="shortcut">
    <string>S</string>
   </property>
  </action>
  <action name="actionHint">
   <property name="text">
    <string>Hint</string>
   </property>
   <property name="shortcut">
    <string>H</string>
   </property>
  </action>
  <action name="actionLevel1">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Level1</string>
   </property>
  </action>
  <action name="actionLevel2">
   <property name="text">
    <string>Level2</string>
   </property>
  </action>
  <action name="actionLevel3">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Level3</string>
   </property>
  </action>
  <action name="actionRank">
   <property name="text">
    <string>Rank</string>
   </property>
  </action>
  <action name="actionInstruction">
   <property name="text">
    <string>Instruction</string>
   </property>
  </action>
  <action name="actionInformation">
   <property name="text">
    <string>Information</string>
   </property>
  </action>
  <action name="actionBackground1">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Background1</string>
   </property>
  </action>
  <action name="actionBackground2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Background2</string>
   </property>
  </action>
  <action name="actionBackground3">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Background3</string>
   </property>
  </action>
  <action name="actionIcon1">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Icon1</string>
   </property>
  </action>
  <action name="actionIcon2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Icon2</string>
   </property>
  </action>
  <action name="actionIcon3">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Icon3</string>
   </property>
  </action>
  <action name="actionMute">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Mute</string>
   </property>
  </action>
  <action name="actionMusic1">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Music1</string>
   </property>
  </action>
  <action name="actionMusic2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Music2</string>
   </property>
  </action>
  <action name="actionLevel4">
   <property name="text">
    <string>Level4</string>
   </property>
  </action>
  <action name="actionLevel5">
   <property name="text">
    <string>Level5</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionMusic3">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Music3</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
